views/dashboard/html-export.php

```diff
@@ -68,4 +74,8 @@
 		<?php submit_button(); ?>
 	</form>
 </div>
-
+<script>
```

Comment:
Nitpick: This is probably fine for now, but if we're on a custom admin screen we should really be enqueuing this JavaScript to load on that screen and not assigning it inline with our markup.

Reply:
Totally agree - actually just forgot to put it in the proper place, will update.